### DISCUSSION TITLE
All browsers opened under ios15 are blank

### DISCUSSION THREAD

**Discussion by 3377** [open-webui/open-webui#4683] [General]
# Bug Report

## Installation Method
Docker
[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]

## Environment

- **Open WebUI Version:** [e.g., v0.3.11]
v0.3.13
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]
NO
- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- IOS 15.1.1
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]
IOS all
**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]
All browsers opened under ios15 are blank, but the computer version is normal.
After logging in under the mobile browser, a prompt box for the latest version will pop up, and then you cannot click I have read or close this window. Refreshing the page will remain blank.

https://github.com/user-attachments/assets/c1eac33d-36a3-4b70-9ecd-245ee57220fb

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by 3377** [comment]
https://github.com/open-webui/open-webui/discussions/2273
This issue has been raised a long time ago, but has not been resolved.

**Comment by 3377** [comment]
反正就是不解决，移到讨论有什么意义。。。

**Comment by 3377** [comment]
Is this bug worth fixing?
@tjbck

**Comment by RocKeD** [comment]
Please fix, I still utilize a combination of new and old hardware. Very useful on older iPads. Please update so it is not needed to purchase new tablets.
